### PR TITLE
Feb 2022 release notes

### PR DESCRIPTION
**Pull request by dannify**

### CODE REVIEW

File: packages/dev/docs/pages/releases/2022-02-11.mdx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: We are back with a new year and a new component! The Contextual Help component has been added to React Spectrum care of @arumsey. This component is handy for showing users more information in forms and other views. We also have some new features including long press support for `MenuTrigger`, additional keyboard support in `Combobox` and `useMove`, and plenty of bug fixes.
```
Comment (snowystinger) [inline packages/dev/docs/pages/releases/2022-02-11.mdx:14]:
```suggestion
description: We are back with a new year and a new component! The Contextual Help component has been added to React Spectrum thanks to @arumsey. This component is handy for showing users more information in forms and other views. We also have some new features including long press support for `MenuTrigger`, additional keyboard support in `Combobox` and `useMove`, and plenty of bug fixes.
```

File: packages/dev/docs/pages/releases/2022-02-11.mdx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+We are back with a new year and a new component! The Contextual Help component has been added to React Spectrum care of [@arumsey](https://github.com/arumsey) 🥳 This component is handy for showing users more information in forms and other views. We also have some new features including long press support for `MenuTrigger`, additional keyboard support in `Combobox` and `useMove`, and plenty of bug fixes.
+
+Thank you to all our contributors who helped us with this release!
+
+
+## New Component
+- Contextual Help
+    - `@react-spectrum/contextualhelp` - [Docs](https://react-spectrum.adobe.com/react-spectrum/ContextualHelp.html)
+## New features
+- **Add support for long press on `MenuTrigger` - [@lishichengyan](https://github.com/lishichengyan) - [PR](https://github.com/adobe/react-spectrum/pull/2678)
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2022-02-11.mdx:29]: extra stars?

Comment (dannify) [reply packages/dev/docs/pages/releases/2022-02-11.mdx:29]: Waiting to see if it is included in release. Will remove once finalized.

File: packages/dev/docs/pages/releases/2022-02-11.mdx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+- Add `useProvider` to @adobe/react-spectrum - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/2769)
+- Support `onKeyDown` in `ComboBox`  - [@sdjustin](https://github.com/sdjustin) - [PR](https://github.com/adobe/react-spectrum/pull/2674)
+- Add modifier keys to `useMove` - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/2794)
+- Add `maxHeight` to `useOverlayPosition` - [@sttwarrior](https://github.com/sttwarrior) - [PR](https://github.com/adobe/react-spectrum/pull/2673)
+- Add `defaultOpen` and `onOpenChange` to `ActionMenu` - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/2728)
+## Fixed
+- Update usage of `start` to use `flex-start` in CSS
+- Fix loss of focus on `NumberField` on Android - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/2127)
+- Update `FocusScope` to restore focus correctly when children change - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/2791)
```
Comment (dannify) [inline packages/dev/docs/pages/releases/2022-02-11.mdx:38]:
```suggestion
- Update `FocusScope` to restore focus correctly when children change - [@kherock](https://github.com/kherock) - [PR](https://github.com/adobe/react-spectrum/pull/2791)
```

File: packages/dev/docs/pages/releases/2022-02-11.mdx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: We are back with a new year and a new component! The Contextual Help component has been added to React Spectrum thanks to @arumsey. This component is handy for showing users more information in forms and other views. We also have some new features including long press support for `MenuTrigger`, additional keyboard support in `Combobox` and `useMove`, and plenty of bug fixes.
+date: 2022-02-11
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2022-02-11.mdx:15]: fix date here and below

File: packages/dev/docs/pages/releases/2022-02-11.mdx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+Thank you to all our contributors who helped us with this release!
+
+
+## New Component
+- Contextual Help
+    - `@react-spectrum/contextualhelp` - [Docs](https://react-spectrum.adobe.com/react-spectrum/ContextualHelp.html)
+## New features
+- Add support for long press on `MenuTrigger` - [@lishichengyan](https://github.com/lishichengyan) - [PR](https://github.com/adobe/react-spectrum/pull/2678)
+- Add `useProvider` to @adobe/react-spectrum - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/2769)
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2022-02-11.mdx:30]: Should we link to the docs for these?

File: packages/dev/docs/pages/releases/2022-02-11.mdx [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+- ColorField beta
+    - [React Spectrum](https://react-spectrum.adobe.com/react-spectrum/ColorField.html)
+    - [React Aria](https://react-spectrum.adobe.com/react-aria/useColorField.html)
+- SearchWithin alpha
+- Calendar alpha
+- DatePicker alpha
+- ListView alpha
+- CardView alpha
+- Card alpha
+
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2022-02-11.mdx:72]: TagGroup?

File: packages/dev/docs/pages/releases/2022-02-15.mdx [hunk 7]
```diff
@@ -0,0 +1,256 @@
...
+## New features
+- Add support for long press on `MenuTrigger` - [@lishichengyan](https://github.com/lishichengyan) - [PR](https://github.com/adobe/react-spectrum/pull/2678) - [Docs](https://react-spectrum.adobe.com/react-spectrum/MenuTrigger.html#long-press)
+- Add `useProvider` to @adobe/react-spectrum - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/2769) - [Docs](https://react-spectrum.adobe.com/react-spectrum/Provider.html#useprovider)
+- Support `onKeyDown` in `ComboBox`  - [@sdjustin](https://github.com/sdjustin) - [PR](https://github.com/adobe/react-spectrum/pull/2674)
+- Add modifier keys to `useMove` - [@snowystinger](https://github.com/snowystinger) - [PR](https://github.com/adobe/react-spectrum/pull/2794)
+- Add `maxHeight` to `useOverlayPosition` - [@sttwarrior](https://github.com/sttwarrior) - [PR](https://github.com/adobe/react-spectrum/pull/2673)
+- Add `defaultOpen` and `onOpenChange` to `ActionMenu` - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/2728)
+## Fixed
+- Update usage of `start` to use `flex-start` in CSS
```
Comment (dannify) [inline packages/dev/docs/pages/releases/2022-02-15.mdx:36]:
```suggestion
- Update usage of `start` to use `flex-start` in CSS - [@msabramo](https://github.com/msabramo) - [PR](https://github.com/adobe/react-spectrum/pull/2695)
```

**Review by devongovett — APPROVED** [review]

**Pull request merged**